Cdk/Config

```diff
@@ -0,0 +1,26 @@
+package.Aws-ec2-imagebuilder-cdk-example = {
```

Comment:
Please remove this file - it is not needed.

Reply:
thank. fix in next version